avocado/core/data_dir.py

```diff
@@ -284,3 +285,55 @@ def clean_tmp_files():
         shutil.rmtree(tmp_dir, ignore_errors=True)
     except OSError:
         pass
+
```

Comment:
There's one very minor (style) issue here: we require two empty lines between functions.  I can fix that while merging it.